### PR TITLE
Empty page and toast

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
In this PR, I have added the following features:
- Toast Notifications (react-toastify)
- Not found page
- Image for empty cart and wishlist page
- Filter bar for mobile
- Rating stars on ProductCard

Please review this PR and suggest

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,13 +4,22 @@ import { Routes, Route } from "react-router-dom";
...
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
 
 import Mockman from "mockman-js";
 
 const App = () => {
   return (
    <>
    <Header />
+   <ToastContainer
```
Comment (arjun-computer-geek) [inline src/App.js:16]: This is the easiest way to show notification

File: src/assets/index.js [hunk 2]
```diff
@@ -17,3 +17,9 @@ export { default as whiteGoldPlate} from "./images/white-plate-gold-border.jpg"
 export { default as comboSet2} from "./images/combo-set-2.jpg";
 export { default as comboSet3} from "./images/combo-set-3.jpg";
```
Comment (arjun-computer-geek) [inline src/assets/index.js:22]: I like most the way you export and import the images and files. Great @ApurvaSawant11

**Review by arjun-computer-geek — APPROVED** [review]
Else look great. Great Folder structures. Easy to understand. Great job @ApurvaSawant11

File: src/components/header/Header.jsx [hunk 3]
```diff
@@ -149,45 +149,85 @@ const Header = () => {
         }
       >
         <div className="flex-row-center drawer-header">
-          <Link to="/products" className="nav-link">
```
Comment (Vishnu-Aithal) [inline src/components/header/Header.jsx:152]: suggestion - Except innerText everything else is same can be mapped from list of a headings

**Review by Vishnu-Aithal — APPROVED** [review]
Very Good Code and Readable.
Good that all you handler functions does not access variables outside, and uses only the arguments.
Good use of Ternary operator chaining

File: src/components/header/Header.jsx (hunk 3 above)
Comment (ApurvaSawant11) [reply src/components/header/Header.jsx:152]: Yeah. I have made the changes. Thanks!

**Pull request merged**